docs/_source/getting_started/installation/configurations/user_management.md

```diff
@@ -42,99 +42,98 @@ rg.init(workspace="my_shared_workspace")
 rg.set_workspace("my_private_workspace")
 ```
 
-### *users.yml*
-
-The above user management model is configured using a YAML file which server maintainers can define before launching a Argilla instance.
-This can be done when launching Argilla from Python or with the provided `docker-compose.yml`. Read below for more details on the different options.
-
 ## Default user
 
-By default, if you don't configure a `users.yml` file, your Argilla instance is pre-configured with the following default user:
+By default, if the Argilla instance has no users, following default admin user will be configured:
 
 - username: `argilla`
 - password: `1234`
 - api_key: `argilla.apikey`
 
-for security reasons we recommend changing at least the password and API key.
+for security reasons we recommend changing at least the password and API key. You can configure
+the default user as follow:
+
+```bash
+python -m argilla.tasks.users.create_default --password changeme --api-key new-api-key
+```
+
 
 :::{note}
 To connect to an old Argilla server using client `>=1.3.0`, you should specify the default user API key `rubrix.apikey`.
 Otherwise, connections will fail with an Unauthorized server error.
 :::
 
-### Override default API key
+## Add new users and workspaces
+
+The above user management model is configured using the Argilla tasks, which server maintainers can define before launching an Argilla instance.
 
-To override the default API key you can set the following environment variable before launching the server:
+### Prepare database
+First of all, we need to be sure database tables and models are up-to-date. This task must be launched when a new version of Argilla is installed
 
 ```bash
-export ARGILLA_LOCAL_AUTH_DEFAULT_APIKEY=new-apikey
+python -m argilla.tasks.database.migrate
 ```
 
+_Note: Is important to launch this tasks prior to other database action._
 
-### Override default password
-
-To override the password, you must set an environment variable that contains an already hashed password.
-You can use `htpasswd` to generate a hashed password:
 
+### Creating an admin user
 ```bash
-htpasswd -nbB "" my-new-password
-:$apr1$n0C4S20a$noG.3yWxH1OIKfFITgzl30
+python -m argilla.tasks.users.create --role admin
 ```
 
-Afterwards, set the environment variable omitting the first `:` character (in our case `$apr1$n0C4...`):
-
+### Creating an annotator user
 ```bash
-export ARGILLA_LOCAL_AUTH_DEFAULT_PASSWORD="<generated_user_password>"
+python -m argilla.tasks.users.create --role annotator
```

Comment:
Missing params

Reply:
I can include them but the CLI task will ask you for all the required parameters